services/src/main/scala/cromwell/services/auth/ecm/EcmConfig.scala

```diff
@@ -3,8 +3,8 @@ package cromwell.services.auth.ecm
 import com.typesafe.config.Config
 import net.ceedubs.ficus.Ficus._
 
-final case class EcmConfig(baseUrl: String)
+final case class EcmConfig(baseUrl: Option[String])
 
 object EcmConfig {
-  def apply(config: Config): Option[EcmConfig] = config.as[Option[String]]("ecm.base-url").map(EcmConfig(_))
+  def apply(config: Config): EcmConfig = EcmConfig(config.as[Option[String]]("ecm.base-url"))
```

Comment:
What if enabled is false?

Reply:
Discussed via Slack but posting here for posterity: Since it is separate from ecm config it is handled separately [here](https://github.com/broadinstitute/cromwell/pull/7392/files#diff-a5fa21faa166c9f13f5872ce92ce2058189460f0c75cef4a3b885d119a50a586R46)